composer.json

```diff
@@ -11,7 +11,7 @@
     ],
     "require": {
         "php": "^7.3 || ^8.0",
-        "phpactor/completion": "~0.4.3",
+        "phpactor/completion": "dev-master as 0.4.5",
```

Comment:
This has to be adjusted when the new release is going to happen.

Reply:
you can change this  to `0.4.5` only,no need for `dev-master`